lib/web_ui/lib/src/engine/text/word_breaker.dart

```diff
@@ -2,7 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-part of engine;
+
+import 'package:ui/src/engine.dart' show clampInt;
```

Comment:
We should eventually find a way to break cycles in our code. Cycles can also increase analysis and compilation times by creating large strongly connected components. We can do this after we get rid of all part files. No rush.